test/integration/doorkeeper_integration_test.rb

```diff
@@ -0,0 +1,9 @@
+require "test_helper"
+
+class DoorkeeperIntegrationTest < ActionDispatch::IntegrationTest
+  test "foo" do
+    visit "/oauth/applications"
+
+    assert_equal 403, page.status_code
+  end
+end
```

Comment:
I think it's usual to have a newline at the end of a file.

Reply:
My bad! Now fixed.